docker/agent/Dockerfile

```diff
@@ -47,14 +47,15 @@ RUN apt-get update \
         libc-ares-dev libjemalloc-dev libsystemd-dev \
         python python3-dev python-setuptools
 
-ENV NGHTTP2_VERSION 1.41.0
+# 
+ENV NGHTTP2_VERSION 1.46.0
 RUN cd /tmp \
     && curl -L "https://github.com/nghttp2/nghttp2/releases/download/v${NGHTTP2_VERSION}/nghttp2-${NGHTTP2_VERSION}.tar.gz" -o "nghttp2-${NGHTTP2_VERSION}.tar.gz" \
     && tar -zxvf "nghttp2-${NGHTTP2_VERSION}.tar.gz" \
     && cd /tmp/nghttp2-$NGHTTP2_VERSION \
     && ./configure \
     && make \
-    && make install
+    && make install || true
```

Comment:
Seems odd, wouldn't we want to fail if install doesn't work?

Reply:
It's failing on a single machine, and I can't explain why. And we don't use this machine to run http2 benchmarks. So for now I am ignoring it.